staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -859,6 +861,13 @@ type ClockOffsetUTC struct {
 // Zone name follows the TZ environment variable format (e.g. 'America/New_York').
 type ClockOffsetTimezone string
 
+// ClockOffsetLocalTime sets the the guest clock to be synchronized to the host's configured timezone when booted, if any.
+type ClockOffsetLocalTime struct {
+	//DisableMigrate determines whether the vm can be migrated,
+	//because it is not known whether other nodes have the same timezone with the source node.
+	DisableMigrate *bool `json:"disableMigrate"`
```

Comment:
I won't expose this field to the KubeVirt API. The VM isn't migratable when this option is set but it isn't something that the user should choose. Try to have a look at other cases when the VM isn't migratable

Reply:
You means if we set clock to Localtime, vm is automatically set to non-migratable? 
Maybe I need to do some subtraction.

Reply:
Yes, it is a consequence of the flag `setlocaltime`. Additionally, once/if we add the scheduling information, then we can migrate on nodes with the same tz and this condition could be removed